_includes/header.html

```diff
@@ -2,9 +2,14 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title }}</a>
+    <div class="navLogo">
+      <a class="site-title" href="{{ site.baseurl }}/"><img src="/images/smLogo.png" alt=""></a>
+    </div>
+
+    <!-- <a class="site-title" href="{{ site.baseurl }}/">{{ site.title }}</a> -->
```

Comment:
In general I'd go ahead and delete this sort of stuff. It'll be in the initial commit if we ever need to go back.